pkg/asset/installconfig/powervs/metadata.go

```diff
@@ -152,23 +152,25 @@ func (m *Metadata) SetDNSInstanceCRN(crn string) {
 
 // GetExistingVPCGateway checks if the VPC is a Permitted Network for the DNS Zone
 func (m *Metadata) GetExistingVPCGateway(ctx context.Context, vpcName string, vpcSubnet string) (string, bool, error) {
+	const colonPercentW = ": %w"
+
 	if vpcName == "" || vpcSubnet == "" {
 		return "", false, nil
 	}
 
 	vpc, err := m.client.GetVPCByName(ctx, vpcName)
 	if err != nil {
-		return "", false, errors.Wrap(err, "failed to get VPC")
+		return colonPercentW, false, fmt.Errorf("failed to get VPC: %w", err)
```

Comment:
The usage here changed because of a new lint error. :(

Reply:
@hamzy I don't see the PR history complaining about that being an empty string. I see the linter complaining about it since it was used multiple times. My concern was that you are returning something other than the empty string. Do you have an example output of the first complaint?

Reply:
```
pkg/asset/installconfig/powervs/metadata.go:161:10: string `: %w` has 4 occurrences, make it a constant (goconst)
```

Reply:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_installer/7544/pull-ci-openshift-installer-master-golint/1708920013651447808/build-log.txt

Reply:
Ah I think I see what you are asking about... let me investigate...

Reply:
@barbacbd I pushed a new version...